app/assets/stylesheets/application.scss

```diff
@@ -11,3 +11,16 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+.navbar-lewagon {
```

Comment:
Hey I dont think this is a good practice you should create your own scss partial